.github/workflows/build-lint-test.yml

```diff
@@ -80,21 +80,17 @@ jobs:
         with:
           go-version-file: 'go.mod'
 
+      - name: Install golangci-lint
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/HEAD/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.1.6
```

Comment:
I guess it all boils down to trust, but installing a dependency over curl with a pinned version and not to a commit sounds worrying to me. 

I would prefer a slower install pinned to a commit and as a source on GitHub.

Reply:
Using the golangci-lint action now 👍